students/rachel_s/session04/mailroom_with_dicts.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import csv
+import os
+import datetime
+
+# Import donor data
+
+donor_info = {}
+
+with open('mailroom/donors.csv') as csv_file:
+    csv_reader = csv.reader(csv_file, delimiter=',')
+    for line in csv_reader:
+        if not line[0].lower() in donor_info:
+            donor_info[line[0].lower()] = [int(line[1])]
+        else:
+            donor_info[line[0].lower()].append(int(line[1]))
```

Comment:
Minor again: Consider replacing lines 14-17 to `donor_info.setdefault(lines[0].lower(), []).append(int(line[1]))`, or better yet use `defaultdict`.